operationsgateway_api/src/config.py

```diff
@@ -115,6 +115,13 @@ class ExportConfig(BaseModel):
     max_filesize_bytes: StrictInt
 
 
+class ObservabilityConfig(BaseModel):
+    """Configuration model class to store export observability details"""
+
+    environment: StrictStr
+    secret_key: StrictStr  # apm key
```

Comment:
Could/should this be a `SecretStr` https://docs.pydantic.dev/2.0/usage/types/secrets/ ?

Reply:
Thanks for flagging this: I've created a new issue in the backlog: https://stfc.atlassian.net/browse/DSEGOG-432